packages/ui-bindings/Agent/agentInterface.tsx

```diff
@@ -19,9 +19,17 @@ fragment agentInterface on Agent {
   type
   image
   note
+   primaryLocation{
```

Comment:
Duplicate code from below where the lat and long is also returned in the same query.  I am removing this one